content/components/segmented-control.mdx

```diff
@@ -0,0 +1,426 @@
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624838-1915681f-1c50-4d0e-94e9-98f3537cf285.png"
+      role="presentation"
+    />
+    <Caption>Change or add icons to indicate whether the choice is selected.</Caption>
+  </Dont>
+</DoDontContainer>
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624832-b9ce99d7-e4e3-46fb-8b24-76133fc6dd90.png"
+      role="presentation"
+    />
+    <Caption>Keep the visual cues in each button consistent.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624835-fc3c7c3b-9f55-4001-9e16-9c7e2071bf8d.png"
+      role="presentation"
+    />
+    <Caption>Don't mix visual cues between buttons.</Caption>
+  </Dont>
+</DoDontContainer>
+
+#### Icon-only buttons
+
+Use icons without a visible text label sparingly, and only with icons that are easily understood by an average GitHub user. When a segment has an icon without a visible text label, the text label should be shown in a tooltip.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624830-b91217ae-7c79-4698-a71a-a482576e5df9.png"
+      role="presentation"
+    />
+    <Caption>Use icons without a visible label when the icon's meaning is obvious.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624831-82387f4d-29ec-4d14-b3d4-edbdc58f6257.png"
+      role="presentation"
+    />
+    <Caption>Don't use obscure icons without a visible label.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Button label text
+
+Segmented control button labels should be nouns or noun phrases that succinctly describe the choice. Button labels may not wrap to multiple lines.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624839-3ea19093-ca19-4081-b20d-70fae6c17271.png"
+      role="presentation"
+    />
+    <Caption>Phrase choice labels as nouns or noun phrases with as few characters as possible.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624840-b689b044-92d6-452b-b36f-524df912f59f.png"
+      role="presentation"
+    />
+    <Caption>Phrase choice labels as an action or with unnecessarily long strings of text.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Contextual information
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <Box as="figcaption" mt="0">
+    <Text as="h4" fontSize={2} fontWeight="bold" m={0}>
+      Label
+    </Text>
+    <Text as="p" mt="0">
+      Segmented controls must have a text label, even if the label is only accessible to screen readers. However, a
+      visible text label is preferred.
```

Comment:
This is a little confusing. Can we list when a visual label is preferred? There are a number of scenarios where the context/location of the SegmentedControl are enough to understand what's it for.

Reply:
> Can we list when a visual label is preferred?

I think we generally prefer visible labels for everything because it adds clarity. cc @alliethu or @ichelsea to keep me correct or add a better explanation.

---

> There are a number of scenarios where the context/location of the SegmentedControl are enough to understand what's it for.

Great point. I'll make sure to directly mention that in this part of the docs